_posts/2017-04-25-the-everchanging-software-process.md

```diff
@@ -0,0 +1,37 @@
+ 
+To be honest, at first I actually was scared as hell of all this horrible freedom at first, because there was so little to lean on. I would be lying if said that there still aren't days when I would just for once be able to just to colour inside the pre-given lines. This is because in most cases if something is going sideways with our project, you pretty much can only look at the mirror. But I've seriously grown to think that this kind of balance between autonomy and responsibility is definitely the most sane way of working, by giving the power and freedom to change pretty much every aspect of the every day work to the people actually doing it.
+ 
+I started to work on the Harja project almost 3 years ago. The project itself is a large web based open-source service ([Harja@GitHub](https://github.com/finnishtransportagency/harja/)) ordered by the [Finnish Transport Agency](http://www.liikennevirasto.fi/web/en) meant for planning and monitoring road, waterway and railway maintenance and care. At first we assumed a pretty much "classic" scrum model as our very basic outline. This was pretty much because most of us we're quire familiar with it and had positive experience in previous projects. Pretty soon we started to notice things here and there that didn't make sense and were starting to feel as a pain, such as over booked sprints, badly defined tasks, review sessions that didn't really give much value.
+ 
+From the very beginning we've thought our processes and ways of working as "snapshot in time", which serves the current needs of the team, project, service and customer. Obviously all of these are volatile by nature and therefore we found that we needed to have a mechanism to meet these changes time and a time again. 
+ 
+For us, one of the key solutions came from regularly held agile retrospectives. Not that this would be any kind of a revolutionary method, but the idea behind it gave us a chance to iteratively monitor and change our way of working. From very early days of our project, we've followed the following mantra:
+1. Reflect on the past period of time: Observer what doesn't work and needs to be changed and what does and has to be protected and nurtured.
+2. Design the changes that you as a team agree that would be beneficial. This phase requires extra care, so that the actions are very thoroughly defined, they have set responsible people and schedule, conditions which have to be met and finally a defined reason why this action will be taken. (This phase is heavily affected by [S.M.A.R.T. goals](https://www.mindtools.com/pages/article/smart-goals.htm))
+3. Take the actions and bring implement them. At this phase the key is to incorporate the changes just you would for example any other development issue. For us this meant that the development actions simply written as a JIRA ticket and placed on our board.
```

Comment:
"bring implement" ylimääräinen bring?

"just you would" => "just like you would"
"for example" ehkä tarpeeton

"actions simply written as a JIRA ticket" => "actions are simply written as JIRA tickets" ?